Technical-Workshop/Presentation/Home Screen/View/CategoryCollectionViewCell.swift

```diff
@@ -21,5 +21,9 @@ class CategoryCollectionViewCell: UICollectionViewCell {
     func configure(categoryItem: CategoryItem){
         categoryTitle.text = categoryItem.title
         categoryImageView.image = UIImage(named: categoryItem.image.rawValue)
+        if categoryItem.title == "Popular" {
```

Comment:
can we manipulate the selected state instead of manually setting the background color